tokens.html

```diff
@@ -0,0 +1,313 @@
+		<dd>True and false, and pairs with similar concepts.
+		<pre><code class="language-javascript">console.log(true === false); // prints false
+console.log(true === !false); // prints true</code></pre></dd>
+
+		<dt><code>number</code></dt>
+		<dd>A numerical value, regardless of base and order, and no matter real or imaginary.
+		<pre><code class="language-python">7
+2147483647
+0o177
+0b100110111
+3
+79228162514264337593543950336
+0o377
+0x100000000
+0xdeadbeef
+3.14
+10.
+.001
+1e100
+3.14e-10
+0e0
+3.14j
+10.j
+10j
+.001j
+1e100j
+3.14e-10j</code></pre></dd>
+
+		<dt><code>string</code></dt>
+		<dd>Literal text, including numbers and symbols and maybe even more special characters.
+		<pre><code class="language-javascript">let greeting = 'Hello World!';</code></pre></dd>
+
+		<dt><code>char</code></dt>
+		<dd>A string that can comprise only a single character, enforced by the language.
+		<pre><code class="language-elm">'A'
+'z'
+'0'
+'-'
+'\t'
+'\u{2728}'</code></pre></dd>
+
+		<dt><code>symbol</code></dt>
+		<dd>A primitive data type found in some languages, can be thought of as an identifier.
+		<pre><code class="language-smalltalk">#myFirstSymbol "#myFirstSymbol is a symbol in Smalltalk and is the same object as all other #myFirstSymbol symbols"</code></pre></dd>
+
+		<dt><code>regex</code></dt>
+		<dd>A regular expression.
+		<pre><code class="language-javascript">let regexPattern = /((?:^|[^$\w\xA0-\uFFFF."'\])\s]|\b(?:return|yield))\s*)\/(?:\[(?:[^\]\\\r\n]|\\.)*\]|\\.|[^/\\\[\r\n])+\/[dgimyus]{0,7}(?=(?:\s|\/\*(?:[^*]|\*(?!\/))*\*\/)*(?:$|[\r\n,.;:})\]]|\/\/))/;</code></pre></dd>
+
+		<dt><code>url</code></dt>
+		<dd>A link to another page or resource.
+		<pre><code class="language-css">body {
+	background: url(foo.png);
+}</code></pre>
+		<pre><code class="language-markdown">[Prism](https://prismjs.com) is a cool syntax highlighter.</code></pre></dd>
+
+		<dt><code>operator</code></dt>
+		<dd>A symbol that represents an action or process, whether it's a mathematical operation, logical operation, and so on.
+		<pre><code class="language-javascript">x = y
+x *= y
+x !== y
+x++
+-y
+x &gt;&gt;&gt; y
+x || y
+z ? x : y</code></pre></dd>
+
+		<dt><code>variable</code></dt>
+		<dd>An object that is assigned and holds a specific value.
+		<pre><code class="language-less">@nice-blue: #5B83AD;
+@light-blue: lighten(@nice-blue, 20%);</code></pre></dd>
+
+		<dt><code>constant</code></dt>
+		<dd>The name of a a variable that cannot be modified later on.
+		<pre><code class="language-javascript">const PI = 3.14159;</code></pre></dd>
+
+		<dt><code>property</code></dt>
+		<dd>An attribute/characteristic.
+		<pre><code class="language-css">body {
+	color: red;
+	line-height: normal;
+}</code></pre>
+		<pre><code class="language-json">{
+	"data": {
+		"labels": [
+			"foo",
+			"bar"
+		],
+		"series": [
+			[ 0, 1, 2, 3 ],
+			[ 0, -4, -8, -12 ]
+		]
+	},
+	// we even support comments
+	"error": null,
+	"status": "Ok"
+}</code></pre></dd>
```

Comment:
This is a little too long IMO. Maybe we could say: 8 lines max per example code block?

```suggestion
		<pre><code class="language-json">{
	"data": {
		"labels": ["foo", "bar"],
	},
	"error": null,
	"status": "Ok"
}</code></pre></dd>
```